snuba/datasets/configuration/events_analytics_platform/storages/eap_items.yaml

```diff
@@ -156,6 +156,17 @@ query_processors:
         max_execution_time: 25
         timeout_before_checking_execution_speed: 0
 
+deletion_settings:
+  is_enabled: 1
```

Comment:
This will allow the product to make delete queries to the existing endpoint, before I had a runtime config that also enabled things to feature gate it - but anyway maybe you could use the `delete_allocation_policies` and set concurrent queries to 0 or something in non `S4S` regions?

Reply:
@onewland okie so technically what I said is still true but since we haven't set up the topic and the topic mapping it will fail. 
https://github.com/getsentry/snuba/blob/6c487c65e44913a9cbf193509cb9dc3ac398795e/snuba/web/bulk_delete_query.py#L50-L52